packages/core/utils/lib/traverse/query-populate.js

```diff
@@ -51,8 +51,7 @@ const populate = traverseFactory()
         if (root !== key) {
           return data;
         }
-
-        return isNil(value) ? root : `${root}.${value}`;
+        return isEmpty(value) ? root : `${root}.${value}`;
```

Comment:
Can you give me a case where this is needed instead of the original line just to make sure I'm understanding the change?

Reply:
this is needed since something sets it to undefined instead of null most likely one of the early returns. So not doing this chance causes all the tests to fail.